helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -14,9 +14,8 @@ extensions:
 
 receivers:
   {{- include "splunk-otel-collector.otelTraceReceivers" . | nindent 2 }}
-  fluentforward:
-    endpoint: 0.0.0.0:8006
 
+  {{- if .Values.metricsEnabled }}
   # Prometheus receiver scraping metrics from the pod itself
   prometheus/agent:
```

Comment:
I think this receiver should be enabled all the time since it's meant to be used for troubleshooting and these metrics are part of APM bundle currently